src/components/Button/Button.tsx

```diff
@@ -104,6 +104,10 @@ const ButtonRoot = styled(MuiButton)(({ theme, variant }) => ({
     /** @TODO confirm that text buttons don't follow the same min width rules */
     minWidth: 'min-content',
     padding: '0px 16px',
+    paddingRight: 0,
+    paddingLeft: 0,
```

Comment:
maybe I'm misreading, but wouldn't this overwrite the padding specified on line 106?

Reply:
ah yes 🤦🏽‍♀️